main.go

```diff
@@ -182,18 +181,11 @@ func generateTargetArtifacts(w http.ResponseWriter, r *http.Request) {
 	name := mux.Vars(r)["name"]
 	plan := r.FormValue("plan")
 
-	helmTargetArtifactTypeValue := "Helm"
-	yamlsTargetArtifactTypeValue := "Yamls"
-	knativeTargetArtifactTypeValue := "Knative"
-
 	re := regexp.MustCompile(`artifacttype:\\s.*`)
```

Comment:
Can we possibly account for scenarios where mistakenly more than one space is left by using say \s* 

Also can we search for [^\s]+ as the string to search for?

Also please chande artifacttype to artifactType. It was a bug from before.

Reply:
Hey, I made some changes, please check if it satisfied the need. I removed one '\' from your regexp pattern, it seems only '\s' should meet the requirement to represent the whitespace from my test. Let me know if you need any updates. Thanks.